src/ontology/cl-edit.owl

```diff
@@ -30663,6 +30664,15 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:29904965"^^xsd:string) o
 AnnotationAssertion(rdfs:label obo:CL_4032001 "reelin GABAergic cortical interneuron")
 EquivalentClasses(obo:CL_4032001 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534) ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000013879)))
 
+# Class: obo:CL_4033014 (peg cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:22911892"^^xsd:string) Annotation(oboInOwl:hasDbXref "http://purl.obolibrary.org/obo/BTO_0005588"^^xsd:string) obo:IAO_0000115 obo:CL_4033014 "A non-ciliated epithelial cell that is part of some oviduct. It is known in some mammals that a peg cell might express EPCAM, CD44, and ITGA6. in vitro, a peg cell has stem-cell-like qualities, including self-renewal capability.")
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033014 "2023-02-02T15:07:12Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:22911892"^^xsd:string) oboInOwl:hasExactSynonym obo:CL_4033014 "intercalary cell")
```

Comment:
Consider adding exact synonyms:
fallopian tube epithelial stem-like cells
FTE stem-like cell
FTESC (syn type: abbreviation)

"Cells marked with the FTESC antigenic profile reside in the same location as the historically described peg cell; therefore, the peg cell is the FTE stem-like cell." PMID:22911892